controller/src/beerocks/bml/bml.h

```diff
@@ -512,6 +512,16 @@ int bml_wfca_controller(BML_CTX ctx, const char *cmd, char *ret_buf, int ret_buf
  */
 int bml_wfca_agent(BML_CTX ctx, const char *cmd, char *ret_buf, int ret_buf_size);
 
+/**
+ * Trigger Topology Discovery Task.
+ *
+ * @param [in] ctx BML Context.
+ * @param [in] al_mac al_mac of the platform. 
+ *
+ * @return BML_RET_OK on success.
+ */
+int bml_trigger_topology_discovery(BML_CTX ctx, const char *al_mac);
```

Comment:
input parameter as char* without specifying the length and doing sanity that it ends with '\0' is risky and prone for errors.
Since the user enters the MAC address as string in the CLI, please convet it to `sMacAddr` there and change all of the APIs to accept `sMacAddr`

Reply:
keeping as is since the conversion to sMacAddr happens in `bml_internal` and the actual char * is received from std::string which is safe since it adds the `\0`.